public/views/partials/lib/commands/user/create/check.liquid

```diff
@@ -0,0 +1,17 @@
+{% liquid
+  assign c = '{ "errors": {}, "valid": true }' | parse_json
+
+  function c = 'modules/core/lib/validations/email', c: c, object: object, field_name: 'email', key: 'validation.email'
+
+  graphql emails_count = 'modules/user/emails_count', email: object.email | dig: 'users', 'total_entries'
```

Comment:
I mentioned the validator function what created for this. Please test it and let me know if something is not working as designed!